src/sources/youtube.js

```diff
@@ -313,29 +328,25 @@ async function loadFrom(query, type) {
   }
 
   if (!config.search.sources.youtube.bypassAgeRestriction)
-    _switchClient(type === 'ytmusic' ? 'ANDROID_MUSIC' : 'ANDROID')
-
+    _switchClient(type === 'ytmusic' ? 'ANDROID_MUSIC' : 'IOS')
+  
   switch (checkURLType(query, type)) {
     case constants.YouTube.video: {
       debugLog('loadtracks', 4, { type: 1, loadType: 'track', sourceName: _getSourceName(type), query })
       
       const identifier = (/v=([^&]+)/.exec(query) || /youtu\.be\/([^?]+)/.exec(query))[1]
 
       const { body: video } = await makeRequest(`https://${_getBaseHostRequest(type)}/youtubei/v1/player`, {
-        headers: {
-          'User-Agent': ytContext.client.userAgent,
-          'X-GOOG-API-FORMAT-VERSION': '2',
-          ..._additionalHeaders
-        },
         body: {
           context: ytContext,
           videoId: identifier,
           contentCheckOk: true,
-          racyCheckOk: true,
-          params: 'CgIQBg'
+          racyCheckOk: true
         },
-        method: 'POST'
-      })
+        method: 'POST',
+        disableBodyCompression: true
+      });
```

Comment:
```suggestion
      })
```

Do not add semicolons